### DISCUSSION TITLE
Switching schools

### DISCUSSION THREAD

**Discussion by guusbuk** [community/community#58526] [GitHub Education]
I'm going to switch schools after summer. How does that work with the student benefits. Do I link a new school (do my free trials reset?), and is it possible to register the new school to a different account?

**Comment by billythekid** [comment]
Hi @guusbuk 

You don't need to do anything. Your student developer pack will remain on your account for 2 years from the date of acceptance. 

If you're at a new school when that runs out you can reapply at that time with your updated information. You should not apply for the pack on two accounts at the same time, even if they're for different schools. An individual (human being) is allowed one pack at a time. You can add additional email addresses to your account at any time of course.

Cheers,
Billy 🤠
